src/Engine/ProtoCore/Utils/ParserUtils.cs

```diff
@@ -140,8 +140,8 @@ private static Core CodeBlockParserCore()
             var core = new Core(new Options())
             {
                 ParsingMode = ParseMode.AllowNonAssignment,
-                IsParsingCodeBlockNode = true,
-                IsParsingPreloadedAssembly = false
+                IsParsingPreloadedAssembly = false,
+                IsParsingInTestMode = true,
```

Comment:
why should this be true here?

Reply:
It's actually a test helper. I'll see if I can refactor it or rename it so that's clear.